obj-c/options-and-metadata.md

```diff
@@ -15,7 +15,7 @@ The data object allows you to provide details about the video and environment th
 All metadata details except for `property_key` are *optional*, however you'll be able to compare and see more interesting results as you include more details.
 
 - Video details (prepended by `video_`) describe the current video that's playing and are all reset automatically when [changing the video](changing-the-video). This metadata would come from your internal CMS or video management system.
-- Player details (prepended by `player_`) describe the player configuration that's being used and should be set whenever monitoring is started on a new player. They do not reset when the video is changed.
+- Player details (prepended by `player_`) describe the player configuration that's being used and should be set whenever monitoring is begun on a new player. They do not reset when the video is changed.
```

Comment:
@ToBeGosu I think this should "is initialized". "is begun" doesn't sound quite right to me.